packages/orchestrator-ui-components/src/graphqlQueries/subscriptionDetailQuery.ts

```diff
@@ -46,10 +46,7 @@ export const GET_SUBSCRIPTION_DETAIL_GRAPHQL_QUERY: TypedDocumentNode<
                     subscriptionInstanceId
                     inUseByRelations
                 }
-                processes(
-                    sortBy: { field: "startedAt", order: ASC }
-                    filterBy: { field: "isTask", value: "false" }
```

Comment:
Why removing the filter?

Reply:
We need the tasks to find the last time the subscription was validated. So I remove the tasks from the subscription workflows in code somewhere else.